documentation/Setup guide.md

```diff
@@ -28,12 +28,12 @@ The Hub Framework requires you to link with Apple's `SystemConfiguration` framew
 
 Each application using the Hub Framework will have one instance of the `HUBManager` class, which manages an instance of the framework. It’s recommended that you create this object early in your application’s lifecycle and keep it somewhere accessible - for example in your `AppDelegate`.
 
-In order to create a `HUBManager` instance, you need to supply implementations of the following 2 protocols:
+There are a few different ways to setup `HUBManager`, each providing different levels of customizability. The simplest way is to use the defaults, and just supply the following two parameters:
 
-- `HUBComponentLayoutManager`: For calculating margins and other layout attributes for components based on their layout traits.
-- `HUBComponentFallbackHandler`: For providing fallback information in case a component model’s component couldn’t be resolved by the framework.
+- `componentMargin`: What margin that should be used in between components.
+- `componentFallbackBlock/Closure`: A block/closure that returns a component to use for fallback, in case a default one couldn't be matched.
```

Comment:
Perhaps 
> in case a default one couldn't be matched

could be clarified with something like

> in case a component identifier with no matching component factory is provided